v3/src/components/graph/utilities/pixi-points.ts

```diff
@@ -72,16 +72,17 @@ export class PixiPoints {
 
   resizeObserver?: ResizeObserver
 
-  activeTransitions = 0
-  currentTransition?: PixiTransition
+  currentlySetupTransition?: PixiTransition
```

Comment:
Simplify name to `currentTransition`?

Reply:
That's what I used before. However, I think `currentTransition` implies it's a transition currently playing, which isn't the case here. This refers to the transition we're setting up via a provided callback. Once it starts, we actually set this property to undefined. Hence, I added `setup` to the name to clarify. 

I'm happy to improve this name, but given this context, do you have any suggestions? Or do you still think that `currentTransition` is better regardless?

Reply:
Good point, but after considering alternatives I'm inclined to think that `currentTransition` still seems best along with a comment that explains what you explained above, i.e. that it's the currently installed/configured transition, not necessarily that it's actively transitioning.

Reply:
@kswenson, I've pushed this change directly to pixi-main branch: https://github.com/concord-consortium/codap/commit/e43bbb299761587ae1efde176e2286bec68135f0